plugins/llvm/llvm_binary.cpp

```diff
@@ -46,59 +46,59 @@ extern "C" {
 
 
     const char* segment_name(const seg::segment* s) {
-        return s->name().c_str();
+        return s->name.c_str();
```

Comment:
sync your master with the upstream, there is no need to review changes that are already in the upstream.